product_docs/docs/pgd/5/admin-tpa/upgrading_major_rolling.mdx

```diff
@@ -6,85 +6,85 @@ deepToC: true
+1. Stop Postgres.
   - `systemctl stop postgres`
-- Install the new Postgres version and bdr_pg_upgrade.
+1. Install the new Postgres version and `bdr_pg_upgrade`.
   - `sudo apt install edb-bdr<pgd_version_number>-pg<postgres_version_number> edb-bdr-utilities`
-- Initialize the new Postgres version.
+1. Initialize the new Postgres version.
   - Make a new directory `/opt/postgres/datanew` to hold the upgraded PGDATA directory
   - Initialize the new version of Postgres in the new directory using the newly installed version's `initdb` 
-- Copy the configuration for the old Postgres version to the configuration for the new Postgres version.
+1. Copy the configuration for the old Postgres version to the configuration for the new Postgres version.
   - Copy the current version's configuration files to the new data directory: `postgresql.conf`, `postgresql.auto.conf`, `pg_hba.conf`, and the whole `conf.d` directory.
-- Rename the PGDATA directories to switch the old version's out and the new version's in.
+1. Rename the PGDATA directories to switch the old versions out and the new versions in.
   - Move `/opt/postgres/data` to `/opt/postgres/dataold` and move `/opt/postgres/datanew` to `/opt/postgres/data`.
-- Double-check that both Postgres instances are not running.
-  - Use `pg_ctl` to stop both `/opt/postgres/dataold` and `/opt/postgres/data`
-- Check that you can upgrade safely.
-  - Use the `--check` option when running `bdr_pg_upgrade`
-- Run the Postgres major version upgrade.
+1. Double-check that both Postgres instances aren't running.
+  - Use `pg_ctl` to stop both `/opt/postgres/dataold` and `/opt/postgres/data`.
+1. Check that you can upgrade safely.
+  - Use the `--check` option when running `bdr_pg_upgrade`.
+1. Run the Postgres major version upgrade.
   - Run `bdr_pg_upgrade` again without the `--check` option.
-- Update the Postgres service configuration to the new version and refresh the system's service manager to apply the changes.
+1. Update the Postgres service configuration to the new version and refresh the system's service manager to apply the changes.
   - Replace the old version number with the new version number in the `postgres.service` file.
     - `sudo sed -i -e 's/<old_version_number>/<new_version_number>/g' /etc/systemd/system/postgres.service`
-- Start Postgres.
+1. Start Postgres.
   - `systemctl start postgres`
-- Validate the new Postgres version. 
-  - Connect to the bdrdb database and run `SELECT * FROM bdr.group_versions_details;`
-- Test the upgrade directly against the newly upgraded node.
-  - Run your test suite against while still SSH'd into the newly upgraded node.
-- Cleanup post-upgrade. 
+1. Validate the new Postgres version. 
+  - Connect to the `bdrdb` database and run `SELECT * FROM bdr.group_versions_details;`.
+1. Test the upgrade directly against the newly upgraded node.
+  - Run your test suite again while still connected by way of SSH into the newly upgraded node.
+1. Clean up post-upgrade. 
```

Comment:
This should have been an ordered list.

You might want to also do numbers under some of these sub-steps.

I think adding the detail makes this a lot harder to read given that all this info is in the details that follow. Just my two cents on this addition.